File: Aprendendo-C/Exercício/soma_vetor.c
#include <stdio.h>

int main(){

    int n;
    double soma, media, cont;
    printf("Quantos numeros você vai digitar? \n");
    scanf("%i",&n);
    double vet[n];

    for(int i = 0;i < n; i++){

    printf("Digite um numero: ");
    scanf("%lf",&vet[i]);
    }

    soma =0;

    printf("Valores: ");
    for(int i = 0;i < n; i++){    
    soma += vet[i];
    cont++;
    printf("%.1lf  ",vet[i]);
    }
    
    media = soma / cont;
    printf("\n Soma: %.1lf\n",soma);
    printf("Media: %.1lf\n",media);


   return 0; 
}

File: Aprendendo-Python/Exercícos/soma_vetor.py
n = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(n)]

for i in range(0,n):
    vet[i] = float(input("Digite um numero: "))

print()
print("Valores = ", end="")
for i in range(0,n):
    print(f" {vet[i]:.1f} ", end="")
print()

soma = 0
for i in range(0,n):
    soma += vet[i]

print(f"Soma =  {soma:.1f}")